HIP/hip-0000-cancun-1.md

```diff
@@ -0,0 +1,115 @@
+---
+hip: <HIP number (this is determined by the HIP editor)>
+title: Add EVM Support for new non-blob Cancun opcodes
+author: Danno Ferrin (@shemnon)
+working-group: // TBD
+type: Standards Track
+category: Core
+needs-council-approval: Yes
+status: Draft
+created: // TBD
+discussions-to: // TBD
+updated: <comma separated list of dates>
+requires: <HIP number(s)>
+replaces: <HIP number(s)>
+superseded-by: <HIP number(s)>
+---
+
+## Abstract
+
+Update the Hedera EVM to add support for new Opcodes found in the Cancun fork of
+Ethereum Mainnet, namely TSTORE, TLOAD, and MCOPY
+
+## Motivation
+
+Hedera's goal of Ethereum Equivalence also includes the requirement to stay
+up-to-date with the current state of Ethereum Mainnet. This HIP addresses new
+opcodes added to the EVM that do not address any features relating to Blobs, or
+the Consensus Layer, or changes to existing opcodes.
+
+## Rationale
+
+The opcodes discussed in this HIP only exist within the EVM and do not interact
+with any novel features of ethereum mainnet. It is expected that future versions
```

Comment:
Q: did you mean Hedera here?
```suggestion
with any novel features of Hedera mainnet. It is expected that future versions
```
or
```suggestion
with any novel features of Hedera services. It is expected that future versions
```

Reply:
I meant ethereum mainnet.  The interactions I was alluding to are things like blob and beacon root.